widget_driver_generator/pubspec.yaml

```diff
@@ -20,6 +20,5 @@ dependencies:
 
 dev_dependencies:
   build_runner:
-  flutter_test:
-    sdk: flutter
   flutter_lints: ^1.0.4
+  test: ^1.22.2
```

Comment:
Because `AnnotatedElement` and `ProvidableField` both import `import 'package:analyzer/dart/element/element.dart';`, which has a transient dependency to dart:mirrors, which is forbidden in flutter, we can't use flutter tests in the generator test, but the dart test library.